src/index.d.ts

```diff
@@ -118,12 +131,11 @@ type TransactionOptions = {
 }
 
 type PositionsOptions = {
+  groups?: Array<string>;
   institutions?: Array<string>;
-  groupIds?: Array<string>;
-  institutionIds?: Array<string>;
-  investmentIds?: Array<string>;
-  includeAssets?: Boolean;
-  includeLiabilities?: Boolean;
+  investments?: Array<string>;
+  assets?: Boolean;
```

Comment:
same here, could be boolean or array? Or maybe we do not have this option at all for positions API